README.md

```diff
@@ -74,13 +74,13 @@ It is often the easiest to move everything depending on the arguments into the `
 
 ### default/example is a string/has extra surrounding quotes
 
-This is caused by a missing `lib.literalExpression` in default/example. Please open a pull request against the source of that option to fix this.
+This is caused by a missing `lib.literalExpression` in default/example. Please open a pull request against the source of that option to fix this. (e.g. [change in nixos-hardware](https://github.com/NixOS/nixos-hardware/pull/1343))
```

Comment:
I think that is the wrong question. I fixed eval in that PR, not quoting of nix Code.